src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -478,21 +478,24 @@ private void getMatchingAttributes() throws IOException {
                     break;
                 }
             }
+            String className;
             MBeanAttributeInfo[] attributeInfos;
-
             try {
+                log.debug("Getting class name for bean: " + beanName);
```

Comment:
note to self: I was going to say these new log lines perhaps didn't add much value, but since we're calling methods on `connection` we can probably trigger an `IOException` with these, so it's probably justified.